src/main/java/org/jenkinsci/modules/windows_slave_installer/WindowsSlaveInstaller.java

```diff
@@ -250,7 +250,7 @@ public AgentURLMacroProvider(@Nonnull LaunchConfiguration launchConfig) {
             
             final String macroValue;
             if (remotingURL != null) {
-                macroValue = "<download from=\"" + remotingURL.toString() + "\" to=\"%BASE%\\slave.jar\"/>";
+                macroValue = "<!-- <download from=\"" + remotingURL.toString() + "\" to=\"%BASE%\\slave.jar\"/> -->";
```

Comment:
I thought the agreement with @daniel-beck was that you only need to do this if the `remotingURL` scheme is not `https`.

(so long as a self signed cert does not blow up the update/service start on the agent...)